extractor/filesystem/language/haskell/cabal/testdata/valid

```diff
@@ -0,0 +1,7 @@
+constraints: any.AC-Angle ==1.0,
+             any.ALUT ==2.4.0.3,
+             any.ANum ==0.2.0.2,
+             any.Agda ==2.6.4.3,
+             transformers-base +orphaninstances,
+             any.Allure ==0.11.0.0,
+             
```

Comment:
There's some whitespace after the last newline in valid and valid_2, remove those
(our internal linters complain if the files don't end in newlines)